### PR TITLE
Day2 notes first draft

### PR DESCRIPTION
**Pull request by dlebauer**
Please edit / comment

### CODE REVIEW

File: content/post/2017-05-27-version-control-and-collaborative-development.Rmd [hunk 1]
```diff
@@ -1,128 +0,0 @@
----
```
Comment (klajbor2) [inline content/post/2017-05-27-version-control-and-collaborative-development.Rmd]: This file is being reverted to before the 641d2e2 merge, is that intentional?

File: content/post/2017-05-27-the-terminal-unix-shell.Rmd [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
```
Comment (klajbor2) [inline content/post/2017-05-27-the-terminal-unix-shell.Rmd]: This file is also being reverted pre 6fa2e99 merge, is this intentional?

**Comment by klajbor2** [comment]
This branch seems to contain files for the day1 notes that are not the latest edits. Specifically, when I checked it out, the day1 notes were from before the 641d2e2 merge of the day1-notes branch. Though perhaps I'm mistaken?

I'm going over the day2 notes now.

File: content/post/2017-05-30-databases-and-sql.Rmd [hunk 3]
```diff
@@ -1,4 +1,20 @@
...
+
+<iframe src="https://docs.google.com/presentation/d/1dQHlodU9PggM7khqlL_OS4P-X-o96D78ihuvuD2Iong/embed?start=false&loop=false&delayms=3000" frameborder="0" width="960" height="749" allowfullscreen="true" mozallowfullscreen="true" webkitallowfullscreen="true"></iframe>
+
+## Spreadsheets and Tables
+
+A database is a generic term for data. Often these are divided into 'relational' and 'non-relational'... We will get to these later. First, lets start with Spreadsheets, which are a simple, and very common format for data.
+
+In fact, so far we have been working with tables.
 
 ## Miscanthus Yields
```
Comment (klajbor2) [inline content/post/2017-05-30-databases-and-sql.Rmd]: I didn't have acces to the link in this section, do the students have access?

File: content/post/2017-05-30-databases-and-sql.Rmd [hunk 4]
```diff
@@ -151,9 +151,61 @@ It can also be easily read by any software or parsed at the command line using t
...
+
+* Each variable forms a column.
+* Each observation forms a row.
+* Each type of observational unit forms a table.
+
+From [tidyr vignette](https://cran.r-project.org/web/packages/tidyr/vignettes/tidy-data.html)
+
+> YOUR TURN Is the Miscanthus dataset tidy? What would we need to do to make it tidy?
+
+## Relational Databases and SQL
```
Comment (klajbor2) [inline content/post/2017-05-30-databases-and-sql.Rmd]: This section was left empty, should we link to some primer on relational databases and SQL?

File: content/post/2017-05-30-databases-and-sql.Rmd [hunk 5]
```diff
@@ -0,0 +1,516 @@
...
+
+The [relational model](https://en.wikipedia.org/wiki/Relational_model) for managing databases consists of representing the data in terms of tuples of relations. Many modern databases use some version of this. Structured Query Language or [SQL](https://en.wikipedia.org/wiki/SQL) is a language implementation based on the relational model, although there are some differences.
+
+### Data Normalization
+
+"Codd's Normal Forms" from (Codd 1970)  
+
+> process of organizing the columns (attributes) and tables (relations) of a relational database to reduce data redundancy and improve data integrity. [Wikipedia - Database normalization](https://en.wikipedia.org/wiki/Database_normalization)
+
+To motivate the use of a relational databases, lets start by looking at a table. Is there redundant information (that violates the )
```
Comment (klajbor2) [inline content/post/2017-05-30-databases-and-sql.Rmd]: What was meant to go inside the parentheses of this line?

**Comment by dlebauer** [comment]
@klajbor2 I got emails with your comments that files from last Friday were reverted. Sounds like a possible error on my part, but I don't see these changes - did you fix this?

**Comment by klajbor2** [comment]
@dlebauer No, it was a mistake on my end. I read something wrong and I was looking at the wrong commits. I noticed it quickly afterwards, but forgot to mention it here.

**Pull request merged**